src/components/content/order/orderStatus/ProcessingStatus.tsx

```diff
@@ -36,6 +36,22 @@ export const ProcessingStatus = (response: ServiceDetailVo, operationType: Opera
         }
     }
 
+    if (operationType === (OperationType.Migrate as OperationType)) {
+        if (response.serviceDeploymentState === ServiceDetailVo.serviceDeploymentState.DESTROY_SUCCESS) {
```

Comment:
we must also display the service details of the migrated service on the new cloud provider. 
Similar to what have in 'Deployment Successful'.